### PR TITLE
Upgrade tchannel @ 2.0.0

### PR DESCRIPTION
**Pull request by Raynos**
All of the features.

r: @jwolski

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -107,8 +107,9 @@ function RingPop(options) {
     this.debugFlags = {};
     this.joinSize = options.joinSize;
     this.pingReqSize = 3;           // ping-req fanout
-    this.pingReqTimeout = 5000;
-    this.pingTimeout = 1500;
+    this.pingReqTimeout = options.pingReqTimeout || 5000;
```
Comment (Raynos) [inline index.js]: new feature: specify options for these things.

File: index.js [hunk 2]
```diff
@@ -256,7 +264,7 @@ RingPop.prototype.bootstrap = function bootstrap(opts, callback) {
             return;
         }
 
-        self.logger.info('ringpop is ready', {
+        self.logger.debug('ringpop is ready', {
```
Comment (Raynos) [inline index.js]: new feature: less logs

File: lib/swim/join-sender.js [hunk 3]
```diff
@@ -251,7 +251,7 @@ JoinCluster.prototype.join = function join(callback) {
 
             self.ringpop.stat('timing', 'join', joinTime);
             self.ringpop.stat('increment', 'join.complete');
-            self.ringpop.logger.info('ringpop join complete', {
+            self.ringpop.logger.debug('ringpop join complete', {
```
Comment (Raynos) [inline lib/swim/join-sender.js]: new feature; less logs

File: package.json [hunk 4]
```diff
@@ -9,7 +9,7 @@
     "Jake Verbaten <raynos2@gmail.com>",
     "Mark Yen <mark@uber.com>"
   ],
-  "version": "9.8.11",
+  "version": "10.0.0-beta1",
```
Comment (Raynos) [inline package.json]: we'll need to change this.

File: package.json [hunk 5]
```diff
@@ -29,6 +29,7 @@
     "farmhash": "^0.2.0",
     "metrics": "^0.1.8",
     "readable-stream": "^1.0.33",
+    "tchannel": "2.0.0",
```
Comment (Raynos) [inline package.json]: we'll remove this and make it a dev dep again soon

**Comment by Raynos** [comment]
travis failed because of TchannelServerListenFailedError: tchannel: listen EADDRINUSE flap.

File: index.js (hunk 1 above)
Comment (jwolski) [inline index.js]: thx

**Comment by jwolski** [comment]
tick-cluster is still a little busted. going to work through that breakage before merging...

**Pull request merged**